CONTRIBUTING.md

```diff
@@ -23,54 +24,85 @@ To keep your forked repository up to date, setup the `upstream` remote to pull i
     $ git remote add upstream https://github.com/void-linux/void-packages.git
     $ git pull --rebase upstream master
 
+This can also be done with the `github-cli` tool:
+
+    $ gh repo fork void-linux/void-packages
+    $ gh repo clone <user>/void-packages
```

Comment:
```suggestion
    $ gh repo fork --clone void-linux/void-packages
```

Reply:
I'm aware of that, but I think keeping it as two commands is good because it's clearer what someone who already has forked the repo can do

Reply:
Ah, that makes sense. But, as a side note, the one-liner would also clone the fork even if it's already been created.

Reply:
does it properly handle having forks in multiple namespaces? e.g. if i had classabbyamp/void-packages and foobar-org/void-packages, will it show a picker or something?

Reply:
I did a quick test and the answer is no. It only takes into account the fork in the main authorized account (classabbyamp/void-packages in your example).

Reply:
ok, I think I prefer the two-step method then. if someone is experienced with github-cli, maybe they can use the one-step method